### DISCUSSION TITLE
Is it possible to use Intel-based Macos-14 runner in an OSS project?

### DISCUSSION THREAD

**Discussion by horenmar** [actions/virtual-environments#10810] [General questions] [answered]
I just tried moving [Catch2](https://github.com/catchorg/Catch2) to newer runners from macos-12, but using `macos-14-large` fails with

> The job was not started because recent account payments have failed or your spending limit needs to be increased. Please check the 'Billing & plans' section in your settings.

I assume that this is because I on the free tier and don't have enabled payments for runners.

Given that plain `macos-14` is now Arm based, is it still possible to get Intel MacOS runners for OSS projects, or is that discontinued?

**Comment by Cheesebaron** [comment]
I think all the `-large` and `-xlarge` runners follow a slightly different pricing scheme and are not available for OSS projects unless you insert credit card.

https://github.com/actions/runner-images?tab=readme-ov-file#available-images

The list here nicely explains what is available and `macos-14` and `macos-14-large` still seems to be intel, while the `macos-14-xlarge` variants are arm64.

Is it because you want to run benchmarks for Catch2 or why is it that you specifically require Intel runners?

**Reply by horenmar** [reply]
`macos-14` is definitely an Arm machine, are you looking at `macos-13`?

And the reason to have both is simple; AppleClang behaves differently when targeting Intel and Arm machines.

**Reply by Cheesebaron** [reply]
Ah yes, you are right. it is the `-large` that is still intel and `macos-14` is arm.

But still, since pricing is different for `-large` and `-xlarge` I don't think you can run on these for free.

**Reply by horenmar** [reply]
Well, yes, I found out that I can't use these for free by the fact that the action errors out.

The question is, is there an alternative Intel Macos-14 runner, or are there plans to have one?

**Comment by vaishalipawar1490** [comment, marked as answer]
Hi @horenmar,
For macOS 14, Intel-based runners like macos-14-large are only available to users with GitHub Team or Enterprise plans, so they aren't free for everyone. If you need free access to Intel-based macOS runners, you can use macos-13 or macos-13-large instead.

**Reply by taysta** [reply]
> Hi @horenmar, For macOS 14, Intel-based runners like macos-14-large are only available to users with GitHub Team or Enterprise plans, so they aren't free for everyone. If you need free access to Intel-based macOS runners, you can use macos-13 or macos-13-large instead.

And what should we do now that macos-13 is depreciated?

**Reply by ghost** [reply]
> > Hi @horenmar, For macOS 14, Intel-based runners like macos-14-large are only available to users with GitHub Team or Enterprise plans, so they aren't free for everyone. If you need free access to Intel-based macOS runners, you can use macos-13 or macos-13-large instead.
> 
> And what should we do now that macos-13 is depreciated?

You can use macos-15-intel.

**Reply by colinandres885-boop** [reply]
golang.org/x/crypto/bcrypt

**Comment by StuartFerguson** [comment]
Will this always be the case and if so when will mac OS 13 be dropped?  I ask this as its the only mac os runner at the moment for free use that supports docker/colima.

**Reply by Cheesebaron** [reply]
https://github.com/actions/runner-images/issues/13046 I guess they are not supported anymore and I assume the images will be dropped soon

**Reply by ghost** [reply]
> #13046 I guess they are not supported anymore and I assume the images will be dropped soon

MacOS 13 image was removed on december 8 
<img width="660" height="1434" alt="IMG_0046" src="https://github.com/user-attachments/assets/d3ec1dfe-e898-4937-bc28-f5c73fa2ca38" />
